src/main/java/core/basesyntax/service/operations/Return.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Product;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Return implements Operation {
```

Comment:
```suggestion
public class ReturnOperation implements Operation {
```
Class names should be noun